src/components/Footer.js

```diff
@@ -25,8 +25,9 @@ const socialMedias = [
   },
 ];
 
+const { muiGridBlockContainerObj } = stylesBase;
 const useStyles = makeStyles({
-  muiGridBlockContainer: stylesBase.muiGridBlockContainer,
+  muiGridBlockContainer: muiGridBlockContainerObj,
```

Comment:
In ES6, you can simplify this:

```javascript
const { muiGridBlockContainer } = stylesBase;
const useStyles = makeStyles({
  muiGridBlockContainer,
 ...
});
```

For object literals if you want the **key** to be the same name as the **variable** that your are assigning as the value, you can just write the name once, without the `key: value` syntax.

Reply:
Yea when I did that it was complaining about a double `const` declaration (second on the rendering function) so I had to renamed on of them.

Do you have a simpler way ?

Reply:
Hrm, that's weird, the second `const` declaration should be scoped to the rendering function, they shouldn't conflict...

What was complaining, eslint? Or was it actually giving you a build error?

Reply:
eslint actually ... Can I ignore it ?

Reply:
Did it say what rule it is referencing?

Reply:
yess
https://eslint.org/docs/rules/no-shadow

Reply:
Maybe just add:

```javascript
//eslint-disable-next-line no-shadow
const {...}
```
That seems  a bit cleaner to me.

Reply:
Okey Dokey